javascript-data-model/order-history.js

```diff
@@ -0,0 +1,88 @@
+
+var orderHistory = ([
+  {
+    orderPlaced: 'Auguest 4, 2020',
+    total: 34.00,
+    shipTo: 'JS Masher',
+    orderNum: '114-3941689-87722232',
+    deliveredDate: 'Delievered Aug,8 2020',
+    deliveryMethod: ' ',
+    item: {
+      description1: ['Javascript for Impatient programmers'],
+      author: ['Rauschmayer Dr Axel'],
+      rtrnWdindow: ['Sep 7,2020'],
+      price: [31.00],
+      buyItAgain: [true],
+      viewItem: [true],
+      productReview: [true],
+      orderRetails: [true],
+      invocie: [true],
+      productSupport: [false]
+    }
```

Comment:
Try making the items an array, since an order can have more than one item in it.

```js
items: [
  {
    //  item data goes here.
  }
]
```